.circleci/config.yml

```diff
@@ -644,15 +670,28 @@ workflows:
           requires:
             - "rpm/sign"
             - "tests/integration-static"
-      - "cleanup/packages-dev":
-          context: falco
+      - "publish/packages-deb-dev":
+          context:
+            - falco
+            - test-infra
           filters:
             tags:
               ignore: /.*/
             branches:
               only: master
           requires:
-            - "publish/packages-dev"
+            - "tests/integration-static"
+      # todo(leogr): the cleanup job is intende to work with bintray, now deprecated,
+      #              do we still need it?
+      # - "cleanup/packages-dev":
+      #     context: falco
+      #     filters:
+      #       tags:
+      #         ignore: /.*/
+      #       branches:
+      #         only: master
+      #     requires:
+      #       - "publish/packages-dev"
```

Comment:
```suggestion
```

We might remove the cleanup phase since we're not gonna have the same storage limitations on the new Falco Infra.

I'd remove comments here, anyways. Cleanup script removal in the future, maybe.